components/ProgressRings/ProgressRings.js

```diff
@@ -0,0 +1,68 @@
+import React, { useMemo } from 'react';
+import { View } from 'react-native';
+import { G, Svg } from 'react-native-svg';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import { connectStyle } from '@shoutem/theme';
+import AnimatedProgressRing from './AnimatedProgressRing';
+import ProgressRing from './ProgressRing';
+
+/**
+ * Component rendering given number of rings, which are filled depending on given
+ * percentage values.
+ * It is possible to either specify definite color for each ring by passing ring.color prop,
+ * or give and array of ring.progressColors and component will then interpolate between given
+ * colors, based on given percentage value.
+ */
+const ProgressRings = ({ rings, size, rotation, children, style }) => {
+  const renderRings = useMemo(
+    () =>
+      rings.map((ring, index) => {
+        const ResolvedRingComponent = ring.progressColors
```

Comment:
Every ring will be animated now

Reply:
Nope, still using both because either can be used

Reply:
ProgressRing can be used by passing definite color, to avoid animations.
AnimatedProgressRing is automatically used when user defines array of `progressColors` parameters for ring and then it's animated - value and color are interpolated.